mailer.js

```diff
@@ -26,6 +27,31 @@ module.exports = function (log) {
     return 'href="' + url + '" style="color: #0095dd; text-decoration: none; font-family: sans-serif;"'
   }
 
+  function constructLocationString (location) {
+    // construct the location string from the location object
+    var locationString = ''
+    if (location) {
+      if (location.city) {
+        locationString += location.city + ', '
+      }
+      locationString += location.country + ' ' + '(estimated)'
+    }
+    return locationString
+  }
+
+  function constructLocalTimeString (timeZone, locale) {
+    // if no timeZone is passed, use 'Etc/GMT'
+    moment.tz.setDefault('Etc/GMT')
```

Comment:
https://en.wikipedia.org/wiki/Coordinated_Universal_Time and https://en.wikipedia.org/wiki/Greenwich_Mean_Time - UTC is generally considered the successor to GMT, and is what timezone offsets are calculated from. Until I read those articles, I always thought GMT observed daylight savings time, but it doesn't.